recipes/gpcal/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "gpcal" %}
+{% set version = "0.1.1.26" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: ca87d0418e51cb9b7334b2d48a6102c2730eeb687e4a6a5f85585e7ea98cda07
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - astropy
+    - matplotlib-base
+    - numpy
+    - pandas
+    - python
+    - scipy
+test:
+  imports:
+    - gpcal
+
+about:
+  home: "https://github.com/jhparkastro/gpcal"
+  license: "GPL-2.0-or-later"
+  license_family: GPL
+  license_file: LICENSE.txt
```

Comment:
It seems this project is not packaging the license, so you will need to add it manually